WalletWasabi/Services/TransactionFactory.cs

```diff
@@ -140,6 +140,8 @@ public TransactionFactory(Network network, KeyManager keyManager, IEnumerable<Sm
 				builder.SetChange(changeHdPubKey.P2wpkhScript);
 			}
 
+			builder.OptInRBF = new Random().NextDouble() < 0.02; // 2% RBF transactions
```

Comment:
Does it work? Don't you have to specify a range here? Something like `NextDouble(0, 1)`?

Reply:
`NextDouble` returns a double between 0 and 0.99999999999999978. `Random` class doesn't provide am overloaded version of `NextDouble` that accept a from-to parameters )we can create one but I am not sure it adds any value)

Reply:
I was just asking, because I am lazy to look it up or test. I believe you.